android/Code-style.md

```diff
@@ -66,23 +66,22 @@ Strings added must not have [unescaped HTML tags](https://github.com/commons-app
 
 String literals should be avoided
 
-## Butterknife
+## View Bindings
 
-Contributors have the option of using [butterknife](https://github.com/JakeWharton/butterknife) for their view bindings. We recommend doing so if you can.
+We have decided to move away from Butterknife and gradually replace its usage with [Kotlin Android Extensions ViewBinding](https://kotlinlang.org/docs/tutorials/android-plugin.html#view-binding). 
 
-### Adding Support
-To add support for butterknife we have added the following dependency in the [app/build.gradle](https://github.com/commons-app/apps-android-commons/blob/master/app/build.gradle) file.
+Note: 
+- Whenever a new Android Component written in Kotlin in added, you should use Kotlin Android Extensions ViewBindings. 
+- Whenever an existing Android component which is written in Java is converted to Kotlin, we should ideally replace Butterknife with Kotlin Android Extensions ViewBinding.
+- If you are simply changing or adding anything in an existing Android component which is written in Java then you can continue using Butterknife. 
 
-```
-dependencies {
-    compile "com.jakewharton:butterknife:$BUTTERKNIFE_VERSION"
-    annotationProcessor "com.jakewharton:butterknife-compiler:$BUTTERKNIFE_VERSION"
-}
-```
+### Butterknife
+
+Contributors have the option of using [butterknife](https://github.com/JakeWharton/butterknife) for their view bindings. We recommend doing so if you can.
```

Comment:
Does this sentence kind of contradict the following sentence found above?

>  We have decided to move away from Butterknife and gradually replace its usage with [Kotlin Android Extensions ViewBinding](https://kotlinlang.org/docs/tutorials/android-plugin.html#view-binding).

Reply:
Do you have the availability to make a PR to delete the recommendation?

Reply:
Sure. Just so I understand correctly. You mean this: 

```suggestion
Contributors have the option of using [butterknife](https://github.com/JakeWharton/butterknife) for their view bindings.
```

Reply:
FYI: I've created #9 with the change.